common/lib/xmodule/xmodule/js/spec/video/video_events_plugin_spec.js

```diff
@@ -78,6 +78,13 @@
                 new_time: 1,
                 type: 'any'
             });
+            expect(state.videoEventsPlugin.emitPlayVideoEvent).toBeTruthy();
+        });
+
+        it('can emit "play_video" event after "seek_video" event ', function() {
+            state.videoEventsPlugin.emitPlayVideoEvent = false;
```

Comment:
@noraiz-anwar  i think this test will always fail because you are setting state.videoEventsPlugin.emitPlayVideoEvent = false; so on expect(state.videoEventsPlugin.emitPlayVideoEvent).toBeTruthy(); it will be false . If  videoPlayer's state is destroyed after each case and re initialized with  this.emitPlayVideoEvent = true; i think we do not need any line regarding state.videoEventsPlugin.emitPlayVideoEvent . because its already been set to true. which is the accepted case that it should emit the state.videoEventsPlugin.emitPlayVideoEvent. so we just need to test expect(state.videoEventsPlugin.emitPlayVideoEvent).toBeTruthy(); is working after the seek or not.